### DISCUSSION TITLE
最后可运行于 Windows 7 上的是哪一个版本？

### DISCUSSION THREAD

**Discussion by DOSforever** [lollipopkit/flutter_server_box#624] [Q&A]
还是所有版本最低要求都是 Windows 8 ？ 或 10？

**Comment by GT-610** [comment]
2024年2月之前的版本支持Windows 7，之后的版本均最低要求64位Windows 10.

**Reply by DOSforever** [reply]
试了下 [Jun 25, 2024 v1.0.992](https://github.com/lollipopkit/flutter_server_box/releases/tag/v1.0.992) 版的，还是不行

**Reply by GT-610** [reply]
> 试了下 [Jun 25, 2024 v1.0.992](https://github.com/lollipopkit/flutter_server_box/releases/tag/v1.0.992) 版的，还是不行

这是2024年6月25版本的，属于2024年2月之后的版本

**Reply by DOSforever** [reply]
看了下，要最接近早于2024年2月的只有 [v1.0.493 on Aug 21, 2023](https://github.com/lollipopkit/flutter_server_box/releases/tag/v1.0.493) 版了，但只有源码，不提供各平台的可执行文件了。
再晚一点的只有 [v1.0.918 on May 25, 2024](https://github.com/lollipopkit/flutter_server_box/releases/tag/v1.0.918) 版，也只有源码。

**Reply by GT-610** [reply]
> 看了下，要最接近早于2024年2月的只有 [v1.0.493 on Aug 21, 2023](https://github.com/lollipopkit/flutter_server_box/releases/tag/v1.0.493) 版了，但只有源码，不提供各平台的可执行文件了。 再晚一点的只有 [v1.0.918 on May 25, 2024](https://github.com/lollipopkit/flutter_server_box/releases/tag/v1.0.918) 版，也只有源码。

更早的版本在 https://cdn.lpkt.cn/serverbox/ 中的“CDN”通道下载

> 看了下，要最接近早于2024年2月的只有 [v1.0.493 on Aug 21, 2023](https://github.com/lollipopkit/flutter_server_box/releases/tag/v1.0.493) 版了，但只有源码，不提供各平台的可执行文件了。 再晚一点的只有 [v1.0.918 on May 25, 2024](https://github.com/lollipopkit/flutter_server_box/releases/tag/v1.0.918) 版，也只有源码。

[CDN](https://cdn.lpkt.cn/serverbox/)仍然保留了一些版本，可以从910版本依次往低版本尝试，看哪个版本Windows7仍然可用。正式结束Win7支持的框架是Flutter 3.22，这个版本的框架与2024年5月14日发布，在此时间段之前的构建（基于Flutter 3.19或更早版本）都是可以在Win7上运行。

不过也请注意，**我们不对过时的版本提供任何支持**。

**Reply by DOSforever** [reply]
在这个地址下 https://cdn.lpkt.cn/serverbox/pkg/?sort=time&order=desc&layout=grid 用于 Windows 版的只有 807,844,871,948 版，试了前三个版本都无法在 Windows 7 下运行，只有 807 版用 VxKex 的兼容 Windows 10 模式可以启动，其它两个版本即便用兼容模式也无法启动。
算了，找不到就找不到吧。谢谢回复。
